src/index.tsx

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable no-console */
 import * as React from 'react';
 import { createRoot } from 'react-dom/client';
+import { PersistGate } from 'redux-persist/integration/react'
+import { Provider } from 'react-redux'
+import store, { persistor } from './state/store/_store';
```

Comment:
We probably want to export from `src/state/store/index.ts` and import from `./state`